src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double topBase;
+    private double bottomBase;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double topBase, double bottomBase,double height) {

```

Comment:
Don't pass expressions in the constructor. Create separate variables and pass them on for better code readability. Also, avoid starting method implementation with an empty line.